Dockerfile

```diff
@@ -70,6 +70,7 @@ ENV CLASSPATH /gatk/gatk.jar:$CLASSPATH
 WORKDIR /gatk
 RUN chmod -R a+rw /gatk
 ENV PATH $CONDA_PATH/envs/gatk/bin:$CONDA_PATH/bin:$PATH
+RUN conda update conda
```

Comment:
Which version of conda do we end up with? Would it be more reproducible to just install that version directly in the base or pin it in some other way (is that possible here)?